doc/source/user_guide/io.rst

```diff
@@ -3933,6 +3934,29 @@ returned, this is equivalent to passing a
 
    store.select('df', "columns=['A', 'B']")
 
+Using level name when querying MultiIndex DataFrame.
```

Comment:
can you make this a sub-section (like using timedelta64), name it Query MultiIndex (also rename Using timedelta64 Query timedelta64)